pkg/exchange/binance/exchange.go

```diff
@@ -1005,16 +1005,13 @@ func (e *Exchange) submitMarginOrder(ctx context.Context, order types.SubmitOrde
 		}
 	}
 
-	// could be IOC or FOK
-	switch order.Type {
-	case types.OrderTypeLimit, types.OrderTypeStopLimit:
```

Comment:
remove the limit order type from the case, and keep the stop limit type in the case